### DISCUSSION TITLE
Additon of key sounds

### DISCUSSION THREAD

**Discussion by 19-ade** [monkeytypegame/monkeytype#2280] [Ideas]
How about we add an option where the user can select from a plethora of different keys and as they type the sound produced will resemble the keys selected by the user . 
Inspiration : [Keyboard simulator ](https://kbs.im/)

**Comment by rizwanmustafa** [comment]
Play sound on click is already implemented. Although, it only contains nk creams and typewriter sounds from the keybaord genre.
